src/Runner.Worker/JobExtension.cs

```diff
@@ -316,6 +316,22 @@ public async Task<List<IStep>> InitializeJob(IExecutionContext jobContext, Pipel
                         }
                     }
 
+                    if (message.Variables.TryGetValue("system.workflowFileFullPath", out VariableValue workflowFileFullPath))
+                    {
+                        context.Output($"uses: {workflowFileFullPath.Value}");
+                        var inputs = message.ContextData["inputs"].AssertDictionary("inputs");
```

Comment:
I guess it works without exception even though there is no entry with key "data" in the ContextData. Maybe cause I am c# newbie? Can you tell me how this can work?

Reply:
Good catch, it appears the inputs key is being populated as a result of `PopulateWorkflowPhaseInputs`. I'll add a null check in.